app/src/dw/AndroidManifest.xml

```diff
@@ -0,0 +1,43 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Did you test whether it's possible to install the dw-flavoured-app and ooniprobe side by side? As far as I know, it is currently not possible to install OONI Probe stable and OONI Probe experimental side by side. While that may be acceptable because it's only us who install the experimental app, not allowing installation side by side will be a bit more confusing for people. That said, also allowing installing side-by-side is weird because we are going to have tow ~ooniprobe apps competing for collecting measurements. Did you spend time thinking about which would be the best way to handle this side by side scenario?